src/ExposureHistoryContext.tsx

```diff
@@ -89,6 +88,16 @@ const ExposureHistoryProvider: FunctionComponent<ExposureHistoryProps> = ({
     })
   }, [])
 
+  const getExposureStatus = useCallback(async () => {
+    const exposureStatus = await StorageUtils.getNewExposuresData()
```

Comment:
```suggestion
    const hasNewExposure = await StorageUtils.getHasNewExposure()
```
A little confused on the naming. It looks, like it wants to be a boolean, but is named as a string or object?